api/tests/test_canteens.py

```diff
@@ -172,3 +172,161 @@ def test_soft_delete(self):
 
         # Model was only soft-deleted but remains in the DB
         self.assertIsNotNone(Canteen.all_objects.get(pk=canteen.id).deletion_date)
+
+    def test_search_single_result(self):
+        CanteenFactory.create(publication_status="published", name="Shiso")
+        CanteenFactory.create(publication_status="published", name="Wasabi")
+        CanteenFactory.create(publication_status="published", name="Mochi")
+        CanteenFactory.create(publication_status="published", name="Umami")
+
+        search_term = "mochi"
```

Comment:
Qu'est-ce que tu penses d'ignorer les accents quand on filtre par texte ? Par exemple, "Cantine rosé" peut-être rechercher avec "rose" même avec "rosé". J'ai fait qqch comme ça pour l'ordre des cantines dans `
`GeneratePosterPage/index.vue`

Reply:
Tout à fait d'accord. Pour ce faire il y a deux options :

1- Utiliser [unaccent](https://www.postgresql.org/docs/9.1/unaccent.html), une extension de Postgres qui enlève les accents. Cette extension est déjà installée de base dans CleverCloud.

2- Maintenir une colonne ASCII (avec [unidecode](https://pypi.org/project/Unidecode/) par exemple) et un index dessus. Ceci est plus performant.

Je me penche pour la première pour la simplicité, le support natif coté DB, et parce que [premature optimization is the root of all evil](https://stackify.com/premature-optimization-evil/) :smile: 

Fait ici : https://github.com/betagouv/ma-cantine/pull/442/commits/b145fdb1d431bbeb11d953a3f7a74455027df60e